### ISSUE TITLE
Macro Bug?

### ISSUE THREAD

**Issue by brunocalado** [Larkinabout/fvtt-challenge-tracker#25]
Hi!

This code is not working for me.
`ChallengeTracker.open(16, 4)`

https://user-images.githubusercontent.com/4999783/189502658-df5d9964-a885-4f9f-8424-b0a50f4c2f33.mp4

I tried with find the culprit. 
![image](https://user-images.githubusercontent.com/4999783/189502704-a235d6e3-dc18-423c-8625-e5829da14956.png)

FVTT 9280
CT 1.1

**Comment by Larkinabout** [comment]
Yeah, that's a bug. I've got a couple of fixes to do, so I'll sort that too for v9 and v10.

In the meantime, you can run this code instead:
```
ChallengeTracker.open({outerTotal: 16, innerTotal: 4})
ChallengeTracker.open({outerTotal: 6, innerTotal: 4})
```

**Comment by Larkinabout** [comment]
Fixed in release 2.1.
